raft.go

```diff
@@ -297,6 +301,7 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		stable:                stable,
 		trans:                 trans,
 		verifyCh:              make(chan *verifyFuture, 64),
+		configurationsCh:      make(chan *configurationsFuture, 64),
```

Comment:
Yeah I was taking the other size - I'll lower it to 8 which should be enough for anything.